packages/plugins/typescript/nhost/jest.config.js

```diff
@@ -0,0 +1 @@
+module.exports = require('../../../../jest.project')({ dirname: __dirname });
```

Comment:
Is this config used (since you're pointing the `--config` flag to `../../../../jest.config.js` in the package.json)?

Reply:
I didn't go deep into how this monorepo is configured, you may be right. I however configured jest the way it is configured in other packages. I suggest we keep it as is for now

Reply:
Makes sense! Let's keep it for now then.